packages/wdio-allure-reporter/src/index.ts

```diff
@@ -222,6 +222,7 @@ class AllureReporter extends WDIOReporter {
 
     onTestFail(test: TestStats | HookStats) {
         if (this._options.useCucumberStepReporter) {
+            attachConsoleLogs(this._consoleOutput, this._allure)
```

Comment:
you don't need it. just move it after line 223 and remove at 243-244

Reply:
@BorisOsipov  THank ypou so much ,

 ```
  if (!this.isAnyTestRunning()) { // is any CASE running

            this.onTestStart(test)
        } 
```

I am not sure what this line was intended to do thats why i kept it after this condition so that it wont be logged twice for onteststart and ontestfail

Reply:
> I am not sure what this line was intended to do 

Haha. Me too. It is ridiculous, but I believe if you will walk through `git blame` you find that it was added by me. I don't know do we really need this now. I think it is not required guard and we can move `attachConsoleLogs` before it.

Reply:
I believe it doesn't break anything for 99% users

Reply:
@christian-bromann what do you think suggest should i move it to the top and remove it from the if condition ?

Reply:
@praveendvd I believe Christian doesn't know is it good or not. The only way to do if you test it for mocha\cucumber and check that is work. You have to test cases when:
- mocha test failed
- mocha after\before (each|all) hook failed
- cucumber step failed
- cucumber after\before step hook failed (if it is in js version)
- wdio after\before hook failed (not sure how it should work at all)

Reply:
mhm .. I would say, let's leave it as is

Reply:
@christian-bromann  I would agree with that as its just one extra line of code